common/txmgr/tracker.go

```diff
@@ -92,6 +93,11 @@ func NewTracker[
 }
 
 func (tr *Tracker[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Start(_ context.Context) (err error) {
+	if disabled {
+		tr.lggr.Info("Abandoned transaction tracking disabled")
+		return nil
+	}
```

Comment:
I don't think this is a safe approach, because later when Close() is called and it tries to `StopOnce`, it will return an error about it not being started. I am also concerned about the other subsystems have still have references to this tracker instance - what happens when they call methods on an unstarted instance?

I think we need to cleanly omit/remove the tracker instance from the production code.

Reply:
Sounds good. I'll remove all start(), close() and other calls to the tracker.